tests/tests.py

```diff
@@ -205,10 +205,11 @@ def test_token_expiry_is_not_extended_within_MIN_REFRESH_INTERVAL(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(original_expiry, AuthToken.objects.get().expires)
 
-    @override_settings(REST_KNOX=no_auto_refresh_knox)
+    #@override_settings(REST_KNOX=no_auto_refresh_knox)
```

Comment:
if you want to use `override_settings` you should not use knox_settings in the login view but `settings.REST_KNOX['TOKEN_LIMIT']` as it is done for `AUTO_REFRESH` setting in auth.py.
Then I would not use `no_auto_refresh_knox` but a new `token_limit_knox` settings.

Reply:
As stated previously I didnt get the override setting to work -- because if I try to evaluate `settings.REST_KNOX['TOKEN_LIMIT']` in a print statement thats fine, but if I try to do it in an `if` statement it becomes a `KeyError`. I can push it if you want to take a look.

Reply:
Ill just paste the code here:

```python
        if settings.REST_KNOX["TOKEN_LIMIT"] is not None:
            if request.user.auth_token_set.filter(expires__gt=timezone.now()).count() >= settings.REST_KNOX["TOKEN_LIMIT"]:
                return Response(
                    {"error": "Maximum amount of tokens allowed per user exceeded."},
                    status=status.HTTP_401_UNAUTHORIZED
                )
```

The first if statement will fail with a `KeyError`, if I instead check `knox_settings['TOKEN_LIMIT']` that will work, but then its always `None` since its a default value.

Reply:
I am away from computer for today, Might look at it after that.